### PR TITLE
docs: add working links to platform support tables

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Add working links back to the Renovate issues on GitHub

## Context:

This might be nice to have, though it makes editing the tables a little bit less easy in future. 😄 

Forgot to mention this idea in PR #10511.

Helps with issue #10446.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/faq.md [hunk 1]
```diff
@@ -41,21 +41,21 @@ If you're self hosting Renovate, use the latest release if possible.
...
-| Cloning repo over SSH selfhosted        | None                                   | #5406                 |
-| Configurable merge strategy and message | None, BitBucket will get support first | #7184                 |
+| Feature name                            | Platform                               | See Renovate issue(s)                                        |
+| --------------------------------------- | -------------------------------------- | ------------------------------------------------------------ |
+| Jira issues                             | BitBucket                              | [#3796](https://github.com/renovatebot/renovate/issues/3796) |
+| Merge trains                            | GitLab                                 | [#5573](https://github.com/renovatebot/renovate/issues/5573) |
+| Cloning repo over SSH selfhosted        | None                                   | [#5406](https://github.com/renovatebot/renovate/issues/5406) |
+| Configurable merge strategy and message | None, BitBucket will get support first | [#7184](https://github.com/renovatebot/renovate/issues/7184) |
```
Comment (HonkingGoose) [inline docs/usage/faq.md:58]: I've just noticed that we closed issue:

- #7184

Do we need to update the table, and if so, how should we update it?

Comment (HonkingGoose) [reply docs/usage/faq.md:58]: In the PR that closes #7184, we have this comment:

> I guess it's just I didn't know how to GitHub! Thanks for the quick education, appreciate it.
> 
> `#10867`, `#10868`, `#10869`, and `#10870` have been created for the remaining platforms.

Should I add the links to those issues into the table?

Comment (viceice) [reply docs/usage/faq.md:58]: yes, please update those tables and add the issue links. Thanks

Comment (HonkingGoose) [reply docs/usage/faq.md]: I've updated the table row description and added links to all the issues mentioned in the comment.

**Review by rarkins — APPROVED** [review]

**Pull request merged**